cypress/integration/pattern/pattern.test.js

```diff
@@ -0,0 +1,175 @@
+import 'cypress-file-upload';
+import { config } from '../../../src/configs';
+import {
+  getAllPatterns,
+  typeValue,
+  addPattern,
+  addPatternError,
+  updateValue,
+  updatePattern,
+  deletePattern,
+  wrongValue
+} from './pattern.variables';
+import statuses from '../../../src/configs/statuses';
+import routes from '../../../src/configs/routes';
+import { patternTranslations } from '../../../src/translations/pattern.translations';
+
+describe('pattern tests', () => {
+  beforeEach(() => {
+    cy.login(Cypress.env('ADMIN_LOGIN'), Cypress.env('ADMIN_PASSWORD'));
+    cy.stubRequest('getAllPatterns', getAllPatterns).as('getAllPatterns');
+    cy.visit('/');
+    cy.get(`[href="${routes.pathToPatterns}"]`).click();
+    cy.wait(3000);
+  });
+
+  it('should be visible and contains values', () => {
+    cy.get('[data-cy=pattern-header]').should('be.visible');
+    cy.get('[data-cy=pattern-header]').contains(
+      config.titles.patternTitles.mainPageTitle
+    );
+    cy.get('[data-cy=add-pattern] .MuiButton-label').should('be.visible');
+    cy.get('[data-cy=Фото]').should('be.visible');
+    cy.get('[data-cy=Фото]').contains(config.tableHeadRowTitles.patterns[0]);
+    cy.get('[data-cy=Назва]').should('be.visible');
+    cy.get('[data-cy=Назва]').contains(config.tableHeadRowTitles.patterns[1]);
+    cy.get('[data-cy="Код матеріалу"]').should('be.visible');
+    cy.get('[data-cy="Код матеріалу"]').contains(
+      config.tableHeadRowTitles.patterns[2]
+    );
+    cy.get('[data-cy=Доступний]').should('be.visible');
+    cy.get('[data-cy=Доступний]').contains(
+      config.tableHeadRowTitles.patterns[3]
+    );
+    cy.get('[data-cy=Дії]').should('be.visible');
+    cy.get('[data-cy=Дії]').contains(config.tableHeadRowTitles.patterns[4]);
+
```

Comment:
I am not sure, but maybe it would be better, to write selectors by using Latin letters.

Reply:
We dynamically pass this values from our config file.